client-cli/src/main.rs

```diff
@@ -51,7 +51,7 @@ pub(crate) fn storage_path() -> String {
 #[inline]
 pub(crate) fn tendermint_url() -> String {
     std::env::var("CRYPTO_CLIENT_TENDERMINT")
-        .unwrap_or_else(|_| "http://localhost:26657/".to_owned())
+        .unwrap_or_else(|_| "http://localhost:26657/websocket".to_owned())
```

Comment:
is the URL correct ? in auto sync, it was `ws://localhost:26657/websocket`?

Reply:
if the URL works with "http://" in the beginning -- for non-breaking changes (so that the client can be used in the same way as before), perhaps it can be assumed to be as before (without the "/websocket" path) and the client would then construct the URL with the "/websocket" path?

Reply:
It works with `http://`. But, I think the correct prefix is `ws://`. Changed it in code as well as in integration tests.